src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/tile.cpp

```diff
@@ -4,32 +4,32 @@
 
 #include <vector>
 
-#include "single_layer_tests/tile.hpp"
+#include "single_op_tests/tile.hpp"
 
-using namespace LayerTestsDefinitions;
+using ov::test::TileLayerTest;
 
 namespace {
 
-const std::vector<InferenceEngine::Precision> netPrecisions = {
-        InferenceEngine::Precision::I8,
-        InferenceEngine::Precision::U8,
-        InferenceEngine::Precision::I32,
-        InferenceEngine::Precision::BF16,
-        InferenceEngine::Precision::FP32
+const std::vector<ov::element::Type> model_types = {
+        ov::element::i8,
+        ov::element::u8,
+        ov::element::i32,
+        ov::element::bf16,
+        ov::element::f32
 };
 
-const std::vector<InferenceEngine::Precision> netTPrecisions = {
-        InferenceEngine::Precision::I64,
-        InferenceEngine::Precision::I16,
-        InferenceEngine::Precision::FP16,
+const std::vector<ov::element::Type> model_t_types = {
+        ov::element::i64,
+        ov::element::i16,
+        ov::element::f16,
 };
 
-const std::vector<std::vector<size_t>> inputShapes = {
-        {2, 3, 4},
-        {1, 1, 1},
+const std::vector<std::vector<ov::Shape>> input_shape_static_3D = {
```

Comment:
Maybe this was the intention?
```suggestion
const std::vector<ov::Shape> input_shape_static_3D = {
```

Reply:
Yes, it was done intentionally to pass to `ov::test::static_shapes_to_test_representation()`